### PR TITLE
css-box-model - Aaron Soukaphay

### PR DESCRIPTION
**Pull request by aaronsoukaphay**
Screen Shot 2023-07-19 at 10 22 18 AM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
```
Comment (shawnkost) [inline css-box-model/styles.css:9]: Missing a `padding-top` & `padding-bottom`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Aaron!

**Comment by aaronsoukaphay** [comment]
Screen Shot 2023-07-19 at 10 34 35 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**